### PR TITLE
skip processing non-objects - usually because the object's been deleted

### PR DESCRIPTION
**Pull request by grf**
Had a small problem with most recent embargo ip code;  care must be taken that the objects identified by the pid are still in existence.

### CODE REVIEW

**Comment by jonathangreen** [comment]
I wonder if these should still be put into the table, but marked as orphaned? So that we don't end up with a slowly growing database of orphaned ids?

File: includes/forms.inc [hunk 1]
```diff
@@ -57,10 +57,15 @@ function islandora_ip_embargo_manage_embargo_list_form($form, &$form_state) {
 
   // Loop thru all the pids, populate the objects and get
   // relationships.
+
   while ($data = $embargoes->fetchObject()) {
     $fedora_object = islandora_object_load($data->pid);
-    $rels = $fedora_object->relationships->get();
 
+    if (! is_object($fedora_object)) {
```
Comment (jordandukart) [inline includes/forms.inc:64]: This really only masks the issue that is happening here. We ideally need to implement `hook_islandora_object_purged` (https://github.com/Islandora/islandora/blob/7.x/islandora.api.php#L273) to check to see if the object is ip embargoed when a purge happens. Furthermore, we should add a new batch to the `hook_cron` or an operation to this batch here https://github.com/Islandora/islandora_ip_embargo/blob/7.x/includes/utilities.inc#L277-L280 to delete all references to no longer existing objects.

**Comment by jonathangreen** [comment]
I'm going to make an issue to clean up the database in JIRA and merge this as is for now.

**Comment by jonathangreen** [comment]
rolled this into another pull here to fix some coding standards issues.
https://github.com/Islandora/islandora_ip_embargo/pull/14

**Pull request closed without merging**